inputs/manager-inputs.yaml

```diff
@@ -29,6 +29,14 @@ inputs:
     type: boolean
     default: false
 
+  local_rest_cert_file:
+    description: Deafult location for local certificate on agents
```

Comment:
update description, no need for "local": .. location of the certificate on the agent